projects/fullcalendar/src/lib/fullcalendar.component.spec.ts

```diff
@@ -1,25 +1,85 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
+import { Component } from '@angular/core';
 import { FullCalendarComponent } from './fullcalendar.component';
+import dayGridPlugin from '@fullcalendar/daygrid';
+
+// wrapper component
+@Component({
+  selector: `full-calendar-test`,
+  template: `<full-calendar
+  [plugins]="plugins"
+  [header]="header"
+  [weekends]="weekends"
+  (viewSkeletonRender)="viewSkeletonRenderEvent($event)"
+  ></full-calendar>`
+})
+class TestHostComponent {
+  plugins = [dayGridPlugin];
+  header: {
+    left: 'prev,next today myCustomButton',
+    center: 'title',
+    right: 'dayGridMonth'
+  };
+  weekends = true;
+  viewSkeletonRender = false;
+  updateWeekendsView() {
+    this.weekends = false;
+  }
+  viewSkeletonRenderEvent(event) {
+    this.viewSkeletonRender = true;
+  }
+}
 
 describe('FullCalendarComponent', () => {
   let component: FullCalendarComponent;
   let fixture: ComponentFixture<FullCalendarComponent>;
 
+  let testComponent: TestHostComponent;
+  let testFixture: ComponentFixture<TestHostComponent>;
+
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ FullCalendarComponent ]
+      declarations: [FullCalendarComponent, TestHostComponent]
     })
-    .compileComponents();
+      .compileComponents();
   }));
-
+  // for test private methods
   beforeEach(() => {
     fixture = TestBed.createComponent(FullCalendarComponent);
     component = fixture.componentInstance;
+    component.plugins = [dayGridPlugin];
     fixture.detectChanges();
   });
+  // for test host
+  beforeEach(() => {
+    testFixture = TestBed.createComponent(TestHostComponent);
+    testComponent = testFixture.componentInstance;
+    testFixture.detectChanges();
+  });
 
   it('should create', () => {
     expect(component).toBeTruthy();
+    expect(fixture.nativeElement.querySelector('.fc-toolbar')).toBeTruthy();
+  });
+  it('should unmount and call destroy', () => {
```

Comment:
please add blank lines in between the `it` statements

Reply:
the test "should unmount and call destroy" does NOT actually test that "destroy" was called. It SHOULD. It only checks that the fc-toolbar is gone.

Make a handler for the `_destroy` event in HostFixture that sets a property called `wasDestroyed` to `true`. Then, for the test, check that `wasDestroyed` is `true`.